vocabularies/Org.OData.Aggregation.V1.json

```diff
@@ -40,10 +40,12 @@
             "$Type": "Aggregation.ApplySupportedType",
             "$AppliesTo": [
                 "EntitySet",
+                "Collection",
                 "ComplexType",
```

Comment:
Why do we need ComplexType? And why not EntityType?

Reply:
@GeraldKrause, can you answer this?